src/include/migraphx/permutation.hpp

```diff
@@ -56,12 +56,12 @@ inline std::vector<int64_t> sort_permutation(const Vector& data, Op op)
 }
 
 /*!
- * Returns the permutation needed to apply to the shape to undo the current permutation
+ * Returns the inverse permutation that could be applied to reverse the inputted permutation
```

Comment:
It doesn't reverse a permutation. Doing `invert_permutation({0,1,2,3})` is not 3,2,1,0, rather its 0,1,2,3.

Reply:
Changing it to "undo" rather than "reverse".